server/meeting_summarization.go

```diff
@@ -100,8 +101,9 @@ func (p *Plugin) createTranscription(recordingFileID string) (*subtitles.Subtitl
 
 func (p *Plugin) newCallRecordingThread(bot *Bot, requestingUser *model.User, recordingPost *model.Post, channel *model.Channel, fileID string) (*model.Post, error) {
 	siteURL := p.API.GetConfig().ServiceSettings.SiteURL
+	localizer := i18n.NewLocalizer(p.i18n, requestingUser.Locale)
 	surePost := &model.Post{
-		Message: fmt.Sprintf("Sure, I will summarize this recording: %s/_redirect/pl/%s\n", *siteURL, recordingPost.Id),
+		Message: fmt.Sprintf(localizer.MustLocalize(&i18n.LocalizeConfig{DefaultMessage: &i18n.Message{ID: "ai.summarize_recording", Other: "Sure, I will summarize this recording: %s/_redirect/pl/%s\n"}}), *siteURL, recordingPost.Id),
```

Comment:
I wonder if it would be better to avoid translation on the server here. We could instead send a prop or another post type that tells the client how to render the post. That way we could do all the translations on the client and the posts wouldn't get stuck in whatever language the user happened to be using at the time.

Reply:
I was thinking exactly the same with that. Anyway, about getting stuck in the language of the user, that is not that bad because the message is expected to be in the user language, so if the user change the language later, what you are going to get is an inconsistent header written on english, and a reply in Spanish (because that header looks like is in the message itself).

Reply:
I just realized that there are a lot of other strings that I wasn't translating, probably 10 more, passing it identifiers and generating the post type, is fine, but can be a bit weird.